Service.Core/Network/Public/Include/Network/Connection.hpp

```diff
@@ -42,17 +42,17 @@ class Connection : public std::enable_shared_from_this<Connection>
 {
 public:
     /** @enum OwnerType
-     *  @brief A connection is "owned" by either a server or a client. /
+     *  @brief A connection is "owned" by either a service or a client. /
         And its behaviour is slightly different between the two.
      */
     enum class OwnerType
```

Comment:
According to the new naming, it is better to rename to the `Working Side`

Reply:
Done.
I totally rephrased it.